test/basicfuns.jl

```diff
@@ -410,3 +410,15 @@ end
     @test axes(s, 1) == OffsetArrays.IdOffsetRange(-2:0)
     @test collect(s) ≈ softmax(1:3)
 end
+
+@testset "cloglog and cexpexp" begin
+    @test cloglog(0) == -Inf
+    @test cloglog(1) == Inf
+    @test cloglog((ℯ - 1) / ℯ) == 0
+    @test cloglog(0.5) == -0.366512920581664327012439158232669469454263447837105263053
```

Comment:
Instead of harcoding constants like the last one, perhaps it would be better to compare to something like
```julia
cloglog_big(x::T) where T = T(log(-log(1 -  BigFloat(x))))
```
with `≈`, using `0.1, 0.2, ..., 0.9` and a bunch of random numbers.